jarvis.py

```diff
@@ -17,7 +17,7 @@
 import speech_recognition as sr
 import wikipedia
 import webbrowser   #Standard Library(No need to install)
-#import os
+import os
 
```

Comment:
forget to import screen_brightness_control without this you can't use these function
so add this line (import screen_brightness_control as sbc)
brfore that go to cmd and import screen_brightness_control